pkg/constants/constants.go

```diff
@@ -168,7 +168,7 @@ const (
 	EtcdDataPath = "/var/lib/etcd"
 
 	// ConfigPath is the path to the downloaded config.
-	ConfigPath = "/run/config.yaml"
+	ConfigPath = "/boot/config.yaml"
```

Comment:
A note for my future self: I wonder if we should call this `machine.yaml` since we always refer to it as "machine config".